titus_isolate/allocate/remote/allocator.py

```diff
@@ -37,6 +41,19 @@ def __init__(self):
         self.__empty_cpu = get_cpu_from_env()
         self.__natural2original_indexing = self.__empty_cpu.get_natural_indexing_2_original_indexing()
         self.__original2natural_indexing = {v: k for k, v in self.__natural2original_indexing.items()}
+        self.__init_sys_cores_data(config_manager)
+
+    def __init_sys_cores_data(self, config_manager):
+        self.__metadata = {}
+        sys_core_ids = config_manager.get_cached_str(SYS_CORE_IDS, None)
+        if sys_core_ids != None:
+            self.__metadata[SYS_CORE_IDS_KEY] = str(sys_core_ids)
+        sys_cores_usage = config_manager.get_cached_str(SYS_CORES_USAGE, None)
+        if sys_cores_usage != None:
```

Comment:
Probably fine for now, but the way this is implemented is that these configs are (node) global. So you couldnt have some cores hard isolate and some soft.

Reply:
That's correct. The core solver supports it, but the API doesnt.